<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки скрипта</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 231, 231);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>501</width>
     <height>98</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QT_min_donation_sum">
    <item>
     <widget class="QLabel" name="label_min_donation_sum">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Минимальная сумма доната для вывода</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_2_min_donation_sum">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Любое число для ограничения или 0 - чтобы выводить донаты с любой суммой</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="slider_min_donation_sum">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QSpinBox" name="spinBox_min_donation_sum">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="singleStep">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>451</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QT_script_options">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_sorting">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Метод сортировки (для подсчёта общей суммы)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_sorting_1">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>По умолчанию (как есть)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_sorting_2">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>По умолчанию (как есть) - в обратном порядке</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_sorting_3">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>По большей сумме</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_sorting_4">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>По меньшей сумме</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_sorting_5">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>По алфавиту</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QT_sorting">
    <item>
     <widget class="QLabel" name="desc_script_options">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Функция скрипта</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_script_options_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Подсчёт общей суммы всех донатов и сортировка</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_script_options_2">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Вывод полной информации о донатах</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 6px; 
    font-size: 14px;
	border-bottom: 2px solid #2671a3;
	border-right: 2px solid #2671a3;

}
QPushButton:hover {
    background-color: #2671a3;
    color: white;
    border: none;
    border-radius: 6px; 
    font-size: 14px;
	border-bottom: 2px solid #2671a3;
	border-right: 2px solid #2671a3;
}
</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #e74c3c;
    color: white;
    border: none;
    border-radius: 6px; 
    font-size: 14px;
	border-bottom: 2px solid #b63a2f;
	border-right: 2px solid #b63a2f;

}
QPushButton:hover {
    background-color: #c0392b;
    color: white;
    border: none;
    border-radius: 6px; 
    font-size: 14px;
	border-bottom: 2px solid #b63a2f;
	border-right: 2px solid #b63a2f;
}

</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>501</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QT_donations_lines_amount">
    <item>
     <widget class="Line" name="line_min_donation_sum">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_1_donations_lines_amount">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кол-во строк с донатами, которое нужно выводить</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_2_donations_lines_amount">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Любое число для ограничения или 0 - чтобы выводить все строки</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QSlider" name="slider_donations_lines_amount">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_donations_lines_amount"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QT_line_ending">
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_line_ending">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Подставить ключевое слово после суммы доната</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Пример: RUB, руб и т.д.</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QLineEdit" name="lineEdit_line_ending"/>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>21</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(255, 255, 0);
alternate-background-color: rgb(0, 0, 127);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>451</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QT_total_sum">
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_total_sum">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Строка в конце списка с общей суммой всех донатов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_total_sum_1">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Вкл</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_total_sum_0">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выкл</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>454</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QT_show_donation_date">
    <item>
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="label_show_donation_date">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Показывать когда пришёл донат: (пример - 3 дня назад)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_show_donation_date_1">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Вкл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btn_show_donation_date_0">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выкл</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #2ecc71;
    color: white;
    border: none;
    border-radius: 6px; 
    font-size: 14px;
	border-bottom: 2px solid #229a54;
	border-right: 2px solid #229a54;

}
QPushButton:hover {
    background-color: #229a54;
    color: white;
    border: none;
    border-radius: 6px; 
    font-size: 14px;
	border-bottom: 2px solid #229a54;
	border-right: 2px solid #229a54;
}
</string>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
